scripts/integration-test.sh

```diff
@@ -1,6 +1,531 @@
+      > processor.log 2>&1 &
+  echo '   processor started'
+
+  sleep 1
+  socorro middleware \
+      --admin.conf=./config/middleware.ini \
+      --database.database_hostname=$database_hostname \
+      --database.database_username=$database_username \
+      --database.database_password=$database_password \
+      --rabbitmq.host=$rmq_host \
+      --rabbitmq.rabbitmq_user=$rmq_user \
+      --rabbitmq.rabbitmq_password=$rmq_password \
+      --rabbitmq.virtual_host=$rmq_virtual_host \
+      --rabbitmq.standard_queue_name=$rmq_normal_queue_name \
+      --rabbitmq.priority_queue_name=$rmq_priority_queue_name \
+      --rabbitmq.reprocessing_queue_name=$rmq_reprocessing_queue_name \
+      --web_server.wsgi_server_class=socorro.webapi.servers.CherryPy \
+      > middleware.log 2>&1 &
+  echo '   middleware started'
+
+  # tell the test routine to use the extra submission test
+  extra_submission_test=1
+  echo " Done."
+}
+
+#------------------------------------------------------------------------------
+# setup and run the collector, processor and middleware
+# The collector will use the traditional wsgi function that can only receive
+# breakpad crashes on the endpoint /submit
+#------------------------------------------------------------------------------
+function start_standard_socorro_apps() {
+  echo "INFO: starting up *** start_standard_socorro_apps *** "
+  echo "      * in this test we're using the older collector capable of accepting only breakpad crashes"
+  echo "      * the collector writes to both the filesystem using FSPermanentStorage & to RabbitMQ using the $rmq_normal_queue_name queue"
+  echo "      * the processor used Processor2015 with the MozillaProcessorAlgorithm2015"
+  socorro collector \
+      --storage.crashstorage_class=socorro.external.crashstorage_base.PolyCrashStorage \
+      --storage.storage_classes='socorro.external.fs.crashstorage.FSPermanentStorage, socorro.external.rabbitmq.crashstorage.RabbitMQCrashStorage' \
+      --storage.storage0.crashstorage_class=socorro.external.fs.crashstorage.FSPermanentStorage \
+      --storage.storage1.crashstorage_class=socorro.external.rabbitmq.crashstorage.RabbitMQCrashStorage \
+      --resource.rabbitmq.host=$rmq_host \
+      --secrets.rabbitmq.rabbitmq_user=$rmq_user \
+      --secrets.rabbitmq.rabbitmq_password=$rmq_password \
+      --resource.rabbitmq.virtual_host=$rmq_virtual_host \
+      --resource.rabbitmq.standard_queue_name=$rmq_normal_queue_name \
+      --resource.rabbitmq.priority_queue_name=$rmq_priority_queue_name \
+      --resource.rabbitmq.reprocessing_queue_name=$rmq_reprocessing_queue_name \
+      --resource.rabbitmq.routing_key=$rmq_normal_queue_name \
+      --resource.rabbitmq.transaction_executor_class=socorro.database.transaction_executor.TransactionExecutor \
+      --web_server.wsgi_server_class=socorro.webapi.servers.CherryPy \
+      --web_server.ip_address=0.0.0.0 \
+      > collector.log 2>&1 &
```

Comment:
Does this get deleted after? 
Above you used the path definition `./submissions` but here you use `collector.log`. The former looks like it's an executable.